infrastructure/kube/keep-prd/tbtc-v2-monitoring/.env.secret

```diff
@@ -0,0 +1,4 @@
+ethereum-url=
+electrum-url=
+sentry-dsn=
+discord-webhook-url=
```

Comment:
Shouldn't we also set `environment` as a secret? Technically, it's not a secret but is set as the value of `ENVIRONMENT` environment variable for the monitoring script.

Reply:
Look at the `configMapGenerator` section in `kustomization.yaml` for `keep-test` and `keep-prd`. The environment is set as part of the generated config map.